wordpress-mobile/WordPress-Aztec-Android
# Issue/184 add keyboard shortcuts

### Fix
Add keyboard shortcuts as described in https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/184.  The shortcuts available are displayed in a dialog shown by pressing `Alt`+`Ctrl`+`H`.  The dialog can not be shown by holding the meta key (i.e. Search or Windows or Command) since the operating system overrides that key to launch search.  Android also overrides the meta key and `/` combination to show system-level shortcuts.  The `Alt`+`Ctrl`+`H` combination was chosen to match Calypso.  On the bright side, the standard editor shortcuts (i.e. copy, cut, paste, and select all) were included for free since `AztecText` extends `EditText`.

### Test
0. Connect external keyboard.
1. Open demonstration app.
2. Press `Alt`+`Ctrl`+`H`.
3. Notice keyboard shortcuts legend.
4. Try keyboard shortcuts.
